kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -82,58 +83,66 @@ impl CodegenBackend for GotocCodegenBackend {
         check_options(tcx.sess, need_metadata_module);
         check_crate_items(&gcx);
 
-        let items = collect_codegen_items(&gcx);
+        let items = with_timer(|| collect_codegen_items(&gcx), "reachability analysis");
```

Comment:
Can we be more specific, e.g. "codegen items reachability analysis"?

Reply:
I changed it to be "codegen reachability analysis". Let me know if that's OK.

Reply:
Sounds good.